src/schema/notifications.ts

```diff
@@ -49,6 +67,87 @@ export class ZimbraNotifications {
 		this.handleFolderNotifications(notification);
 		this.handleConversationNotifications(notification);
 		this.handleMessageNotifications(notification);
+		this.handleContactNotifications(notification);
+	};
+
+	private handleContactNotifications = (notification: Notification) => {
+		const items = itemsForKey(notification, 'cn');
+		if (items) {
+			let searchResponse: any = {};
+			items.forEach((i: any) => {
+				const item = normalizeContact(i);
+				const foldersMap: any = {
+					'7': 'Contacts',
```

Comment:
I think we can make this more generalized by reading the folderId from cache and checking it's name:

```js
const defaultFolderName = 'Contacts';
const folder = this.cache.readFragment({
	id: `Folder:${item.folderId}`,
	fragment: gql`
		fragment folderName${item.folderId} on Folder {
			name
		}
	`
});
const folderName = folder && folder.name || defaultFolderName;